core/templates/pages/exploration-player-page/exploration-player-page-root.component.html

```diff
@@ -18,7 +18,7 @@
   </oppia-base-content>
 </oppia-angular-root>
 
-<link rel="stylesheet" href="/third_party/static/guppy-f509e1/build/guppy-default.min.css">
+<link rel="stylesheet" href="/node_modules/guppy-dev/build/guppy-default.min.css">
```

Comment:
I would recommend that this be put as an import statement in the files which require guppy. Adding CSS links to index.html like files is not the best approach as we are moving towards a unified entry in our app.

Reply:
removed. thanks @srijanreddy98